velox/expression/tests/SimpleFunctionTest.cpp

```diff
@@ -1609,4 +1609,43 @@ TEST_F(SimpleFunctionTest, callNullFreeNoThrow) {
   EXPECT_EQ(2, result);
 }
 
+template <typename TExec>
+struct DecimalPlusValueFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExec);
+
+  template <typename A, typename B>
+  void initialize(
+      const std::vector<TypePtr>& inputTypes,
+      const core::QueryConfig& /*config*/,
+      const A* /*a*/,
+      const B* /*b*/) {
+    scale_ = getDecimalPrecisionScale(*inputTypes[0]).second;
+  }
+
+  template <typename R, typename A, typename B>
+  void call(R& out, A a, B b) {
+    out = a + b * DecimalUtil::kPowersOfTen[scale_];
+  }
+
+ private:
+  int8_t scale_;
+};
+
+TEST_F(SimpleFunctionTest, helpMessage) {
+  const std::string functionName = "decimal_plus_value";
+  registerFunction<
+      DecimalPlusValueFunction,
+      LongDecimal<P1, S1>,
+      LongDecimal<P1, S1>,
+      int64_t>({functionName});
+  auto function = exec::simpleFunctions().resolveFunction(
+      functionName, {DECIMAL(20, 2), BIGINT()});
+  EXPECT_TRUE(function.has_value());
+  EXPECT_EQ(
+      function.value().helpMessage(functionName),
+      "FunctionName: decimal_plus_value\nSignature argument types:\nDECIMAL(I1,I5), BIGINT\n"
```

Comment:
I find the new message hard to read. What problem are you trying to solve?

Reply:
Before this PR, we don't print the `Metadata` physical argument and result types, and other fields like `priority_` and `defaultNullBehavior_` that user may wish to know.
If we use std::cout to print it, we will get
```
FunctionName: decimal_plus_value
Signature argument types:
DECIMAL(I1,I5), BIGINT
Physical argument types:
HUGEINT, BIGINT
Physical result types:
HUGEINT
Priority:999997
DefaultNullBehavior:true
```

Reply:
This message might be helpful to developers, but not users. 

Can we keep existing message as well, but introduce a separate method for debugging?

Reply:
Ok, can I name the new method `toString`?

Reply:
Sure, this should work.